com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1350,8 +1349,15 @@ private void SyncTime()
 #endif
         }
 
-        private readonly List<NetworkObject> m_ObservedObjects = new List<NetworkObject>();
-
+        /// <summary>
+        /// Server Side: Handles the approval of a client
+        /// </summary>
+        /// <param name="ownerClientId"></param>
```

Comment:
I'm not a big fan of having empty `<param></param>` blocks.
If we don't want to fill them in individually (which I also think that's mostly the case), can we just get rid of empty param blocks boilerplate?